README.md

```diff
@@ -1 +1,70 @@
-# Math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly format your entire README using [the Microverse template](https://github.com/microverseinc/readme-template) and edit the sections with relevant information for this project.